declarative-schedule-doc.md

```diff
@@ -339,6 +340,90 @@ scenarios:
 And on top of that you can use aliases to avoid duplication. For more details, check output
 [official documentation of openQA](http://open.qa/docs/)
 
+### 5. Use YAML_SCHEDULE_DEFAULT and YAML_SCHEDULE_FLOWS to YAML schedule
+
+
+We have 3 variables to help YAML scheduler to avoid conditional schedule and repetition. They are YAML_SCHEDULE_DEFAULT, YAML_SCHEDULE_FLOWS, YAML_SCHEDULE.
+
+Please be noted that conditional schedule does not work with the new scheduler that uses DEFAULT and FLOWS.
```

Comment:
we should not mention that is a "new scheduler" it is just a feature on it, you can use the old path or the new path, this is just for not scaring anybody :) that everything is compatible:
s/with the new scheduler that uses DEFAULT/when scheduler is using DEFAULT/

Reply:
changed.